### DISCUSSION TITLE
Inventory block

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14462] [How to]
how to activate the inventory block in widgets?
![image](https://user-images.githubusercontent.com/54880467/112520478-766d2100-8dc1-11eb-8a42-a288b3ad7836.png)

**Comment by rskrzypczak** [comment]
This option will not be available until version 6.2

**Reply by rioshaz** [reply]
thanks for ur response mate ....
can u help me with :

is it possible to prefetch discount directly from the product when creating a sales invoice..
if not how can it be done,,
can i create a custom field "discounts" in products and can it be directly linked to discounts in advance fields ....like unit price, subunit auto fill ?
possible ?

pls help
